install_pim/manual/crontab_tasks_ee.rst.inc

```diff
@@ -24,12 +24,13 @@ You have to add these 10 commands to your local installation:
     php /path/to/installation/pim-enterprise-standard/bin/console akeneo:reference-entity:refresh-records --all --env=prod # re-index all reference entities
     php /path/to/installation/pim-enterprise-standard/bin/console pim:volume:aggregate --env=prod # recalculates the number of product values
     php /path/to/installation/pim-enterprise-standard/bin/console pimee:data-quality-insights:schedule-periodic-tasks --env=prod # schedule consolidation and purge of data quality insights related data
+    php /path/to/installation/pim-enterprise-standard/bin/console pimee:franklin-insights:quality-highlights:push-structure-and-products --env=prod #push structure and products data in order to compute the Quality Highlights metrics
```

Comment:
could you differentiate the franklin commands and the regular pim commands (as I did in the 3.2 PR no longer needed https://github.com/akeneo/pim-docs/pull/1285/files#diff-51b17d77a778295676a0b0ffd42f0883R32)

It is more clear that the franklin CRONs are not mandatory for every pim install